src/GUI/SSMSToolbox/SSMSToolbox.csproj

```diff
@@ -560,7 +560,7 @@
       <HintPath>..\lib\v140\SQLEditors.dll</HintPath>
     </Reference>
     <Reference Include="SQLiteScripting">
-      <HintPath>..\lib\SQLiteScripting.dll</HintPath>
+      <HintPath>..\..\API\SQLiteScripting\bin\$(Configuration)\SQLiteScripting.dll</HintPath>
```

Comment:
No, the files are built from the API project, and copied to the lib folder

Reply:
I notice that you do this manually; is there a reason why it's not automatically done by the build?

I see `copytogui.cmd` but it doesn't seem like it's in the right place.

Reply:
Sorry, the project is not really properly setup for OSS :-(